cmd/backend/main.go

```diff
@@ -59,17 +60,23 @@ func main() {
 	log.SetOutput(f)
 	log.SetFormatter(&log.JSONFormatter{})
 
+	// Day Information Struct Init
+	dayInfo := DayInformationNew()
```

Comment:
Is it better to have `day.NewDayInfo` rather than `NewDayInfo` by importing slightly differently. I know for some functions its for identification/clarity of source package but in this case is it self-explainority - I'm unsure what the best practice is.

Reply:
This would affect imports for the rest of packages so holding off for that ATM.

Reply:
I'd stick to `day.NewDayInfo` personally... maybe that will change with experience but for now I think better to keep it as `day.NewDayInfo`